src/main/scala/gitbucket/core/controller/api/ApiGitReferenceControllerBase.scala

```diff
@@ -17,6 +17,15 @@ trait ApiGitReferenceControllerBase extends ControllerBase {
    * https://developer.github.com/v3/git/refs/#get-a-reference
    */
   get("/api/v3/repos/:owner/:repository/git/ref/*")(referrersOnly { repository =>
+    getRef()
+  })
+
+  // Some versions of GHE support this path
+  get("/api/v3/repos/:owner/:repository/git/refs/*")(referrersOnly { repository =>
```

Comment:
nit: Would it better to log warning here? Something like `logger.warn("git/refs/ endpoint may not be compatible with GitHub API v3. Consider using git/ref/ endpoint instead")` ?

Reply:
Thanks! Added a warn log to /refs.